mesos/cli/cmds/ssh.py

```diff
@@ -33,6 +33,9 @@
     help="""Name of the task."""
 ).completer = completion_helpers.task
 
+parser.add_argument(
```

Comment:
As there are no arguments for this that are mesos specific yet, I'd like to just pass all the options directly to ssh itself. That way, you don't need to remember to pass in a special arg.